gear-programs/vft-manager/app/src/services/mod.rs

```diff
@@ -306,6 +398,10 @@ where
         &mut self,
         msg_id: MessageId,
     ) -> Result<(), Error> {
+        if self.state().is_paused {
```

Comment:
perhaps it's worth to extract function something like `ensure_running() -> Result<(), Error>` so call it `ensure_running()?;`